Content.Shared/Chat/V2/Types.cs

```diff
@@ -84,11 +38,51 @@ public string Message
         set;
     }
 
-    /// <summary>
-    /// The type of sent message.
-    /// </summary>
-    public MessageType Type
+    public void SetId(uint id)
     {
-        get;
+        if (Id != 0)
+        {
+            return;
+        }
+
+        Id = id;
     }
 }
+
+// TODO: These three channel enums can be migrated to YAML instead of being defined here.
```

Comment:
THESE ARE GOING INTO YAML WHEN SOMEONE CODES IT. It'll be the same as radio channels.